db/migrate/20211118113021_add_login_to_user_table.rb

```diff
@@ -0,0 +1,5 @@
+class AddLoginToUserTable < ActiveRecord::Migration[6.0]
+  def change
+    add_column :users, :login, :string
```

Comment:
You must add an index to keep `login` unique.
We also need to add, inside the User model, a validation to verify the uniqueness of the login field.

Reply:
@syl-p

Reply:
Pas eu encore le temps de travailler dessus.